eland/operations.py

```diff
@@ -800,6 +800,34 @@ def quantile(
         else:
             return df if is_dataframe else df.transpose().iloc[0]
 
+    def unique(self, query_compiler: "QueryCompiler") -> pd.Series:
+
+        query_params, _ = self._resolve_tasks(query_compiler)
```

Comment:
So all the lines are organized conceptually let's do:

```python
query_params, _ = self._resolve_tasks(...)
body = Query(query_params.query)

fields = ...all_source_fields()
assert len(fields) == 1  # Unique is only for eland.Series
field = fields[0]
bucket_key = f"unique_{fields[0].column}"

...
```

Then we can replace all `fields[0]` with `field` below too.